Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserComponent.cpp

```diff
@@ -86,6 +87,8 @@ namespace AzToolsFramework
                     HandleFileInfoNotification(buffer, bufferSize);
                 });
             }
+
+            m_inspectorWidget = new AssetBrowserEntityInspectorWidget();
```

Comment:
Was this for debugging? It seems like this line should be removed, since the `AssetBrowserEntityInspectorWidget` is already being registered as a dockable tool?